### PR TITLE
TFAC-749 Black Equity Images and Cause Data

### PR DESCRIPTION
**Pull request by amaliwanag2**
putting this up here to reveal that we need some ux tweaks, mostly around images

### CODE REVIEW

File: src/data/causes/blackEquity.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      },
+      "Financials": {
+        "title": "Transparent Financials",
+        "text": "Every tab does make an impact!  Check out how much we've raised together.",
+        "buttonText": "See All Financials",
+        "ctaImg": "../../img/blackEquity/financials.png",
+        "q1Img": "../../img/cats/q1.png",
+        "q2Img": "../../img/cats/q2.png",
+        "q3Img": "../../img/cats/q3.png",
+        "q4Img": "../../img/cats/q4.png"
```
Comment (kmjennison) [inline src/data/causes/blackEquity.json:84]: Need the globe images for these:
https://www.figma.com/file/WwF1hgTbUCCVNiT7dA5TM1/TF-Black-Equity?node-id=0%3A23

These will be the generic doodles. TeamSeas will use them too.

File: src/data/causes/blackEquity.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            "question": "How does opening tabs raise money for charity?",
+            "answer": "Every time you open a new tab, there will be a few ads in the right corner of your home screen. These ads generate the revenue that funds our charitable impact. And the best part? It's completely free!"
+          },
+          {
+            "question": "What type of images can I expect on my new tab page?",
+            "answer": "Each day our Tabbers are welcomed with breathtaking images captured by Black photographers. See the collection [here](https://unsplash.com/collections/l26tWeG5IJw/black-equity)."
+          },
+          {
+            "question": "What nonprofit does Tab for Black Equity support?",
+            "answer": "Tab for Black Equity supports [The Bail Project] (https://bailproject.org/) and [Thurgood Marshall College Fund] (https://www.tmcf.org/?gclid=CjwKCAiA5t-OBhByEiwAhR-hmzFLgTNoslU9vYn6NAWGC-XBM5zm2kOeuNLPQdo7yFMlEv0PixPbpRoCKEcQAvD_BwE). We are proud to support these incredible organizations that work to uplift their communities towards paths of empowerment and opportunity."
```
Comment (kmjennison) [inline src/data/causes/blackEquity.json:99]: I think the space in the markdown links will be an issue?

File: src/data/causes/blackEquity.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          },
+          {
+            "question": "How can I find out about new updates and announcements?",
+            "answer": "Stay in touch with us on social media! Follow us @tabforacause on [TikTok] (https://www.tiktok.com/@tabforacause), [Instagram] (https://www.instagram.com/tabforacause/), [Twitter] (https://twitter.com/TabForACause), and [Facebook] (https://www.facebook.com/TabForACause)."
```
Comment (kmjennison) [inline src/data/causes/blackEquity.json:103]: Same: space in the markdown links

File: src/components/V4HomePage.js [hunk 4]
```diff
@@ -125,10 +126,12 @@ const HomepageWrapper = ({
           <Mission missionData={missionData} causeId={causeId} />
           <SecuritySection securityData={Security} />
           <FinancialsComponent financialsData={Financials} />
-          <EndorsementsComponent
-            endorsementsData={Endorsements}
-            causeId={causeId}
-          />
+          {Endorsements ? (
```
Comment (kmjennison) [inline src/components/V4HomePage.js:129]: I don't think this works. In this case, we want to optionally hide only the big quote (not all endorsements), so maybe key on Endorsments.quote? Please add a simple test, too. Feel free to break out into a separate PR if you want.

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/components/V4HomePage.js (hunk 4 above)
Comment (jedtan) [reply src/components/V4HomePage.js:129]: I didn't understand what part to make optional 😂, will adjust.

**Review by kmjennison — APPROVED** [review]
LGTM 🦊

**Pull request merged**